### PR TITLE
Lesson_admin branch #2 merge

### PR DESCRIPTION
**Pull request by bigkeios**
This pull request is for merging these implementations into master: 
- admin edit, delete lessons
- seed.rb for generate lessons data

### CODE REVIEW

File: app/views/lessons/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        <%= f.text_field :name, class: "form-control" %>
+      </div>
+
+      <div class= "form-group">
+        <%= f.label :objective %>
+        <%= f.text_area :objective, class: "form-control" %>
+      </div>
+      
+      <%= f.submit yield(:button_text), class:"btn btn-primary"%>
+  <% end %>
```
Comment (huannd-0028) [inline app/views/lessons/edit.html.erb]: indent

File: app/views/lessons/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  </div>
+</div>
+
```
Comment (huannd-0028) [inline app/views/lessons/edit.html.erb]: remove line

**Comment by bigkeios** [comment]
Fixed convention errors

**Comment by huannd-0028** [comment]
Ok

**Pull request merged**